packages/calypso-e2e/src/lib/pages/marketing-page.ts

```diff
@@ -32,16 +33,7 @@ export class MarketingPage extends BaseContainer {
 	 * @param {Page} page Underlying page on which the actions take place.
 	 */
 	constructor( page: Page ) {
-		super( page, selectors.content );
-	}
-
-	/**
-	 * Post-initialization steps when creating an instance of this object.
-	 *
-	 * @returns {Promise<void>} No return value.
-	 */
-	async _postInit(): Promise< void > {
-		await this.page.waitForSelector( selectors.navtabsList );
+		super( page );
```

Comment:
I think the constructor can be omitted if it's only calling `super` as it's inherited from the `BaseContainer` class.